src/cloud/onboarding/onboarding-tasks.md

```diff
@@ -7,66 +7,80 @@ functional_areas:
 
-The _Project Owner_  is the person in your business or finance organization that manages payments and other business-related issues for the {{site.data.var.ece}} account. The Project Owner serves as the point of contact with Magento.
+The _License Owner_  is the person in your business or finance organization that manages payments and other business-related issues for the {{site.data.var.ece}} account. This person serves as the point of contact with Magento.
 
-To quickly onboard your project so you can develop your site for live deployment, you need to complete specific set up steps and onboarding tasks. Typically, the Project Owner begins the process by securing Admin access and creating Technical Admin users that can help with set up, customization, and development work.
+To quickly onboard your project, so you can begin developing your site for live deployment, you must complete specific set up steps and onboarding tasks. Typically, the License Owner begins the process by securing Admin access and creating Technical Admin users that can help with set up, customization, and development work.
 
-## Project portal
+{:.bs-callout-info}
+If you need to change the License owner on your account, you must submit a Magento Support ticket.
 
-Once you have an account, you can use the Magento Project Portal to manage the project for your {{site.data.var.ece}} store. The portal provides a Getting Started guide and an interactive workflow that helps project administrators, business users, and developers with the tasks required to develop, build, test, and launch your site.
+## Sign up for a Cloud account {#cloud-first-acct}
 
-**Open the Magento Project Portal:**
+Don't have a {{site.data.var.ece}} account yet? Contact [Magento Sales][]. We will create your account and send you a welcome email that provides instructions to access the project.
 
-1. Log in to your Magento account.
+The person who signs up for a {{site.data.var.ece}} account is referred to as the _License Owner_.
 
-1. On the My Account page, click the Magento tab to see the projects in your account.
+## Your welcome e-mail {#email}
 
-1. **Click View Project Page** in the [Projects section](https://cloud.magento.com/cloud/project/subscriptions/).
+Magento sends a welcome email to the License Owner using the address that was provided during the sign up process. The email contains a link to access your {{site.data.var.ece}} project so you can complete initial project set up.
 
-## Sign up for a {{site.data.var.ece}} account {#cloud-first-acct}
+You can also access your project by [logging in to your account][your {{site.data.var.ece}} account].
 
-Don't have a {{site.data.var.ece}} account yet? Contact [Magento Sales](https://magento.com/explore/contact-sales). We will create your account and send you a welcome email that provides instructions to access the project.
+## Cloud Project portal
```

Comment:
Is it Cloud Project or Magento Project? Maybe use a variable, like Magento Commerce Cloud Project...

Reply:
That's the title that CSM uses, and Onboarding UI still used as well.